### PR TITLE
[snapshot] Create minimal detection_rules package

### PR DESCRIPTION
**Pull request by rw-access**
This is the start of a `detection_rules` package.
Related to https://github.com/elastic/security-team/issues/17

Includes
* `kibana/rules/rule-*.json`: Individual rules as separate files
* `kibana/rules/changelog.json`: Complete changelog for all rules in the package
* `manifest.yml`: Specifies the package metadata, uses a non-`integration` package type and marks the package as experimental.

When the format is stabilized and the package is ready to be promoted, a follow on PR will be created to [elastic/package-spec](https://github.com/elastic/package-spec) to add the specification for the `kibana/rules/` folder.

```json
{
  "name": "detection_rules",
  "title": "Detection rules",
  "version": "0.0.1-dev.1",
  "release": "experimental",
  "description": "Rules for the detection engine in the Security application.",
  "type": "rules",
  "download": "/epr/detection_rules/detection_rules-0.0.1-dev.1.zip",
  "path": "/package/detection_rules/0.0.1-dev.1",
  "format_version": "1.0.0",
  "readme": "/package/detection_rules/0.0.1-dev.1/docs/README.md",
  "license": "basic",
  "categories": [
    "security"
  ],
  "conditions": {
    "kibana.version": "^7.12.0"
  },
  "assets": [
    "/package/detection_rules/0.0.1-dev.1/manifest.yml",
    "/package/detection_rules/0.0.1-dev.1/docs/README.md",
    "/package/detection_rules/0.0.1-dev.1/kibana/rules/CHANGELOG.json",
    "/package/detection_rules/0.0.1-dev.1/kibana/rules/rule-a9cb3641-ff4b-4cdc-a063-b4b8d02a67c7.json",
    "/package/detection_rules/0.0.1-dev.1/kibana/rules/rule-ce64d965-6cb0-466d-b74f-8d2c76f47f05.json"
  ],
  "owner": {
    "github": "elastic/protections"
  }
}

```

### CODE REVIEW

File: packages/detection_rules/0.0.1-dev.1/kibana/rules/CHANGELOG.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "1": ["Created \"New ActiveSyncAllowedDeviceID Added via PowerShell\" rule to detect new ActiveSync devices registered via PowerShell."],
+  },
+  "a9cb3641-ff4b-4cdc-a063-b4b8d02a67c7": {
+    "1": ["Created \"IPSEC NAT Traversal Port Activity\" rule to detect IPSEC activity that traverses a NAT"],
+    "2": ["Added dataset to network rules"],
+    "3": ["Remove rule timelines"],
+    "4": ["Add Threat Detection Category"],
+    "5": ["Remove deprecated ATT&CK technique"]
+  }
+}
```
Comment (spong) [inline packages/detection_rules/0.0.1-dev.1/kibana/rules/CHANGELOG.json:12]: You're gonna want a new line here I think.

**Comment by ruflin** [comment]
@rw-access Great to see progress on this. As soon as this stabilises and you get to the package-spec work, check out our change proposal flow: https://github.com/elastic/package-spec/blob/master/CONTRIBUTING.md#change-proposals

File: packages/detection_rules/0.0.1-dev.1/kibana/rules/CHANGELOG.json (hunk 1 above)
Comment (rw-access) [reply packages/detection_rules/0.0.1-dev.1/kibana/rules/CHANGELOG.json]: 1ac3c1ec585b6a0971bf2a07032b3abfd79e0eb5

**Comment by rw-access** [comment]
elastic/package-spec is up to date. a lot has changed and this is an integration package now, so I'm going to update the test package and publish this first to elastic/integrations.

**Pull request closed without merging**